docs/learning-powershell/README.md

```diff
@@ -1,12 +1,12 @@
 Learning PowerShell
 ====
 
-Whether you're a Developer, a DevOps or an IT Professional, this doc will help you getting started with PowerShell.
+Whether you're a developer, a DevOps or an IT professional, this document will help you getting started with PowerShell.
 In this document we'll cover the following:
-installing PowerShell, samples walkthrough, PowerShell editor, debugger, testing tools and a map book for experienced bash users to get started with PowerShell faster.
+Installing PowerShell, samples walkthrough, PowerShell editor, debugger, testing tools, and a map book for experienced Bash users to get started with PowerShell faster.
```

Comment:
Personally, I'd split each **comma-segment** into a **bullet-line** (dropping the `and`).
Something like:

 In this document we'll cover the following:
* Installing PowerShell
* Walking through samples
* PowerShell editor
* debugger
* testing tools
* a map book for experienced Bash users to get started with PowerShell faster.

No matter how you  look at it, that last `map book` line should be reworked beyond this love...